src/dialog.jsx

```diff
@@ -232,72 +226,14 @@ const DialogInline = React.createClass({
     };
   },
 
-
-  _getAction(actionJSON) {
-    warning(false, `using actionsJSON is deprecated on Dialog, please provide an array of
- buttons, or any other components instead. For more information please refer to documentations.`);
-    const props = {
-      secondary: true,
-      onClick: actionJSON.onClick,
-      onTouchTap: () => {
-        if (actionJSON.onTouchTap) {
-          actionJSON.onTouchTap.call(undefined);
-        }
-        if (!(actionJSON.onClick || actionJSON.onTouchTap)) {
-          this._requestClose(true);
-        }
-      },
-      label: actionJSON.text,
-      style: {
-        marginRight: 8,
-      },
-    };
-
-    if (actionJSON.ref) {
-      props.ref = actionJSON.ref;
-      props.keyboardFocused = actionJSON.ref === this.props.actionFocus;
-    }
-    if (actionJSON.id) {
-      props.id = actionJSON.id;
-    }
-
-    return (
-      <FlatButton {...props} />
-    );
-  },
-
   _getActionObjects(actions) {
```

Comment:
We should be able to rewrite this method like this:

```js
  _getActionObjects(actions) {
    return React.Children.map(actions, action => React.isValidElement(action));
  },
```
Since it's a pretty simply function too, maybe we can consider just doing `React.Children.map(actions, action => React.isValidElement(action));` directly in the two places we're calling `_getActionObjects`. Just one less method on the class.